nuxt.config.ts

```diff
@@ -242,6 +242,9 @@ export default defineNuxtConfig({
         'style-src': ['\'self\'', '\'unsafe-inline\''],
         'upgrade-insecure-requests': true,
       },
+      permissionsPolicy: {
+        fullscreen: ['\'self\'', 'https:', 'http:'],
```

Comment:
Does it need to be so permissive? I'll let others review this one in that case.

Reply:
FYI：it's syncs with contentSecurityPolicy.media-src

Reply:
Ah, I see.